src/token.rs

```diff
@@ -31,8 +31,12 @@ impl Default for Namespace {
 /// Creates a new token and saves it into the database.
 ///
 /// Returns created token.
-pub async fn save(context: &Context, namespace: Namespace, foreign_id: Option<ChatId>) -> String {
-    let token = dc_create_id();
+pub async fn save(
+    context: &Context,
+    namespace: Namespace,
+    foreign_id: Option<ChatId>,
+    token: String,
+) -> String {
```

Comment:
is there any point in still returning the token now?  seems a weird api

Reply:
no there is no point, i just forgot it :) thanks for pointing out. i now resultified save() and also updated the documentation.